sound/soc/codecs/max98373-sdw.c

```diff
@@ -211,9 +211,6 @@ static bool max98373_readable_register(struct device *dev, unsigned int reg)
 static bool max98373_volatile_reg(struct device *dev, unsigned int reg)
 {
 	switch (reg) {
-	case MAX98373_R2054_MEAS_ADC_PVDD_CH_READBACK:
-	case MAX98373_R2055_MEAS_ADC_THERM_CH_READBACK:
-	case MAX98373_R20B6_BDE_CUR_STATE_READBACK:
```

Comment:
the commit message states "Patch removes volatile attribute of such registers."

That doesn't seem right, it no longer exposes those registers?

Reply:
We should not remove those registers if they are volatile.

Reply:
Notice that Amp temperature value can be read for calibration in factory test and boot time.., so these registers should be exposed. We need to find some way to avoid errors during suspend..
Any suggestions or inputs please ?